shell/platform/darwin/ios/framework/Source/FlutterTextInputPluginTest.mm

```diff
@@ -60,6 +60,8 @@ @interface FlutterSecureTextInputView : FlutterTextInputView
 
 @interface FlutterTextInputPlugin ()
 @property(nonatomic, assign) FlutterTextInputView* activeView;
+@property(nonatomic, assign) UIView* keyboardViewContainer;
```

Comment:
```suggestion
@property(nonatomic, weak) UIView* keyboardViewContainer;
```

Reply:
@cyanglaz i could be wrong, but iirc the standard practice is just `assign` when exposing the property in test header (similar to the `activeView` properly above). `weak` is confusing since it's actually a `strong` property.

Reply:
Oh. I didn't realize it is only exposing the property. I'm not aware of the standard practice tho but since activeView also used assign, I will take your word for it :)

Reply:
@cyanglaz and I just discussed offline - it should be `readonly` here, since your unit test does not need the setter. 

```
@property(nonatomic, readonly) UIView* keyboardViewContainer;
```

Reply:
Gotcha! That works for me, just went through and made the changes.